app/src/containers/MenuMobile.js

```diff
@@ -0,0 +1,15 @@
+import { connect } from 'react-redux';
+import MenuMobile from '../components/Shared/MenuMobile';
+import { setVisibleMenu } from '../actions/appearence';
+
+const mapStateToProps = (state) => ({
+  menuVisible: state.appearance.menuVisible
```

Comment:
Let's merge it as it is for now (if that's the only blocking point) and improve it later. We need a bit of refactoring as the desktop and mobile menu don't use the same "architecture". We'll be able to remove the redux state after the refactoring.